.gitignore

```diff
@@ -63,3 +63,8 @@ fastlane/Preview.html
 fastlane/screenshots
```

Comment:
Please remove this unnecessary ignores from the file to avoid issues in the future.